L10nSnapshotTests/L10nSnapshotTests.swift

```diff
@@ -6,88 +6,54 @@ import XCTest
 
 class L10nSnapshotTests: L10nBaseSnapshotTests {
     func test02DefaultTopSites() {
+        navigator.goto(HomePanelsScreen)
         snapshot("02DefaultTopSites-01")
     }
 
     func test03MenuOnTopSites() {
-        let app = XCUIApplication()
-        app.buttons["TabToolbar.menuButton"].tap()
+        navigator.goto(NewTabScreen)
+        navigator.goto(BrowserTabMenu)
         snapshot("03MenuOnTopSites-01")
-        app.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.25)).tap()
-
-        loadWebPage(url: "about:blank", waitForLoadToFinish: false)
-        sleep(2)
-        app.buttons["TabToolbar.menuButton"].tap()
-        snapshot("10MenuOnWebPage-01")
-        app.otherElements["MenuViewController.menuView"].swipeLeft()
-        snapshot("10MenuOnWebPage-02")
-        app.coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.25)).tap()
-
-        app.buttons["URLBarView.tabsButton"].tap()
-        app.buttons["TabTrayController.menuButton"].tap()
-        snapshot("10MenuOnTabsTray-02")
     }
 
-    func test04Foo() {
-        let app = XCUIApplication()
-        loadWebPage(url: "http://wopr.norad.org/~sarentz/fxios/testpages/index.html", waitForOtherElementWithAriaLabel: "body")
-        app.buttons["TabToolbar.menuButton"].tap()
-        snapshot("10MenuOnWebPage-01")
-        app.otherElements["MenuViewController.menuView"].swipeLeft()
-        snapshot("10MenuOnWebPage-02")
+    func scroll(_ table: XCUIElement, eachScreen: (Int) -> ()) {
+        // TODO implement steadily scrolling up the screen
+        // hint: find the last cell that is hittable, and swipe up until it isn't
+        eachScreen(0)
     }
 
     func test04Settings() {
-        let app = XCUIApplication()
-        app.buttons["TabToolbar.menuButton"].tap()
-        app.otherElements["MenuViewController.menuView"].swipeLeft()
-        app.cells["SettingsMenuItem"].tap()
-        snapshot("04SettingsTopLevel")
-
-        // TODO Scroll through the settings and make a screenshot of every page
-
-        // Screenshot all the settings that have a separate page
-        for cellName in ["Search", "NewTab", "Homepage", "Logins"] {
-            app.tables["AppSettingsTableViewController.tableView"].cells[cellName].tap()
-            snapshot("04Settings-\(cellName)")
-            app.navigationBars.element(boundBy: 0).buttons.element(boundBy: 0).tap()
+        let table = app.tables.element(boundBy: 0)
```

Comment:
/me c'mon Apple. there is no better function signature they could have come up with to get element by index?

Reply:
Am thinking about a project just for `XCUIElement` extensions.